### PR TITLE
feature/multiple-proxies

### PR DESCRIPTION
**Pull request by IdoJS**
[multiple proxies in one line](https://github.com/scalecube/scalecube-js/issues/83)

### CODE REVIEW

File: packages/scalecube-microservice/tests/integration/Errors/createProxies.spec.ts [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+      And     a microservice instance
+    When      creating a serviceDefinition with the 'serviceName'
+              | type       | 'serviceName' |
+              | array	     | []            |
+              | object	   | {}            |
+              | boolean	   | true          |
+              | boolean	   | false         |
+              | number	   | 10            |
+              | null	     | null          |
+              | undefined	 | undefined     |
```
Comment (vvpyshnenko) [inline packages/scalecube-microservice/tests/integration/Errors/createProxies.spec.ts]: remove undefined

File: packages/scalecube-microservice/src/Proxy/createProxy.ts [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+          isServiceAvailable(serviceDefinition).then(() => resolve({ proxy }));
+        } catch (e) {
+          reject(e);
+        }
+      });
+    } else {
+      try {
+        proxiesMap[proxyName] = createProxy({ serviceDefinition, router, microserviceContext });
+      } catch (e) {
+        proxiesMap[proxyName] = e;
```
Comment (vvpyshnenko) [inline packages/scalecube-microservice/src/Proxy/createProxy.ts]: why does it swallow errors for 'isAsync: false' case

File: packages/scalecube-microservice/src/api/ProxiesMap.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+/**
+ * Map of generic proxyName and a Promise to the proxy
+ */
+export default interface ProxiesMap {
+  [proxyName: string]: Promise<any> | any;
```
Comment (vvpyshnenko) [inline packages/scalecube-microservice/src/api/ProxiesMap.ts]: for readability better  Promise<msProxy> than Promise<any>
use type alias

Comment (IdoJS) [reply packages/scalecube-microservice/src/api/ProxiesMap.ts]:
```suggestion
  [proxyName: string]: Promise<msProxy> | msProxy;

  type msProxy = any;
  
```

**Pull request merged**